templates/crate/source.html

```diff
@@ -21,6 +21,10 @@
     {{ navigation::package_navigation(metadata=file_list.metadata, active_tab="source") }}
 {%- endblock header -%}
 
+{% block body_classes %}
+    {% if file_content %}flex{% endif %}
+{% endblock body_classes %}
```

Comment:
```suggestion
{%- block body_classes -%}
    {%- if file_content %}flex{% endif -%}
{%- endblock body_classes -%}
```

to drop the extra whitespace

Reply:
What is the difference between the two btw?

Reply:
`-` makes it remove all whitespace on that side of the embedded content

Reply:
Thanks!